src/Middleware/WorkerMiddleware.php

```diff
@@ -97,7 +119,33 @@ public function __invoke(
         /** @var ResponseInterface $response */
         $response = $pipeline($request, $response);
 
-        return $response->withHeader('X-HANDLED-BY', 'ZfrEbWorker');
+        return $response->withHeader('X-Handled-By', 'ZfrEbWorker');
```

Comment:
Just for consistency with how uppercasing is done by AWS. Should not cause any BC as if I remember getHeader is supposed to be case-insensitive.